src/gui/updater/ocupdater.cpp

```diff
@@ -303,7 +323,7 @@ void NSISUpdater::versionInfoArrived(const UpdateInfo &info)
             showDialog(info);
         }
         if (!url.isEmpty()) {
-            _targetFile = cfg.configPath() + url.mid(url.lastIndexOf('/'));
+            _targetFile = cfg.configPath() + url.mid(url.lastIndexOf('/')+1);
```

Comment:
So this was broken all along...

Reply:
Yup, but no biggie. Windows usually handles doubled / just fine. Just msiexec is *very* picky about filenames.